utils/releaser/src/main.rs

```diff
@@ -10,11 +11,14 @@ use structopt::StructOpt;
 enum Releaser {
     DeployServer,
     ReleaseApple,
+    ReleaseAndroid,
 }
 
-fn main() {
+#[tokio::main]
+async fn main() {
     match Releaser::from_args() {
         Releaser::DeployServer => server::deploy_server(),
         Releaser::ReleaseApple => apple::release_apple(&Github::env(), &AppStore::env()),
+        Releaser::ReleaseAndroid => android::release_android(&Github::env()).await,
```

Comment:
You should make an `Android::env()` in secrets, so that, I know rust doesn't use it and only gradle will, but this will allow us to fail much earlier if someone's machine is not setup properly. In the `env()` function you create you can also do some basic validations that assert that whatever file they're providing via env vars actually exists.